module/applications/actor/base-sheet.mjs

```diff
@@ -402,7 +402,7 @@ export default class ActorSheet5e extends ActorSheet {
       const physical = [];
       if ( data.bypasses?.size ) {
         values = values.filter(t => {
-          if ( !CONFIG.DND5E.physicalDamageTypes[t] ) return true;
+          if ( !globalThis.CONFIG.DND5E.physicalDamageTypes[t] ) return true;
```

Comment:
Not sure the `globalThis` prefix is necessary to access `CONFIG` here, or below.

Reply:
It’s the only way I could find that would fix the `no-loop-func` ESLint warning that was being incorrectly thrown here. The other option would be to manually annotate it to skip that warning.

Reply:
Honestly, I might vote to just turn off that warning entirely. I don't think I've ever had it flag something that wasn't a false positive. What do you think?

Reply:
I have no objections to that :)